demos/continuous_batching/README.md

```diff
@@ -232,7 +233,7 @@ pip3 install openai
 from openai import OpenAI
 
 client = OpenAI(
-  base_url="http://localhost:8000/v3",
+  base_url="http://localhost:11401/v3",
```

Comment:
why this port?

Reply:
This is leftover from debugging, will remove it